src/NzbDrone.Core/Indexers/Definitions/IPTorrents.cs

```diff
@@ -364,7 +364,7 @@ public IPTorrentsSettings()
         [FieldDefinition(1, Label = "Base Url", Type = FieldType.Select, SelectOptionsProviderAction = "getUrls", HelpText = "Select which baseurl Prowlarr will use for requests to the site")]
         public string BaseUrl { get; set; }
 
-        [FieldDefinition(2, Label = "Cookie", HelpText = "Site Cookie")]
+        [FieldDefinition(2, Label = "Cookie", HelpText = "Site Cookie - see wiki for more info")]
```

Comment:
If we have a link, use “HelpLink” prop so that we link to it

Reply:
suggested code work?
![image](https://user-images.githubusercontent.com/55419169/129456501-b4272aaf-ab9e-443a-947c-284aa6628397.png)

is it possible to do a code block for the cookie formatting?